runtime/sema/type.go

```diff
@@ -5526,12 +5526,12 @@ func (m *Member) IsStorable(results map[*Member]bool) (result bool) {
 		return result
 	}
 
-	// Temporarily assume the member is non-storable while it's type
+	// Temporarily assume the member is storable while it's type
 	// is checked for storability. If a recursive call occurs,
 	// the check for an existing result will prevent infinite recursion
 	// and the result will be correctly false
```

Comment:
Can probs drop this last line of the comment, as it no longer guarantees any result on a recursive check.

Reply:
Good point! Done in bc31767